atdgen/test/github.atd

```diff
@@ -0,0 +1,1254 @@
+type error = {
```

Comment:
Could you insert a comment that explains where these definitions come from? (which are impressive, btw)